tests/snapshots/tests.contrib.langchain.test_langchain.test_openai_service_name[mysvc-None].json

```diff
@@ -0,0 +1,112 @@
+[[
+  {
+    "name": "langchain.request",
+    "service": "mysvc",
```

Comment:
Should langchain take on mysvc with v0? I thought only web frameworks were supposed to do this?

Reply:
yes, fixing

Reply:
This took a second look.  The answer is actually no, due to explicit design decisions for the open AI integration.

In particular, the openAI integration design specifies implement peer-service like behavior, so the module does some explicit work (like using `int_service`, setting measured tags, and overriding `requests.request`) to get desired behavior.  Since openAI has been out since April, we need to maintain this pattern.